plc4go/protocols/modbus/readwrite/model/DataItem.go

```diff
@@ -24,30 +24,48 @@ import (
 	"github.com/apache/plc4x/plc4go/spi/utils"
 	"github.com/apache/plc4x/plc4go/spi/values"
 	"github.com/pkg/errors"
+	"math"
 )
 
 // Code generated by code-generation. DO NOT EDIT.
```

Comment:
I hope you read this line and didn't manually edit this file ... because the next time you'll run the full maven build these changes are going to be replaced with the old values.